internal/install/install_run_test.go

```
@@ -756,22 +756,22 @@ func TestRun_SectionAwareOverwritePreservesUserEntries(t *testing.T) {
 
 	issuesPath := filepath.Join(root, "docs", "agent-layer", "ISSUES.md")
-	initial, err := os.ReadFile(issuesPath)
+	initial, err := os.ReadFile(issuesPath) // #nosec G304 -- path is constructed from test-controlled inputs.
 	if err != nil {
 		t.Fatalf("read issues: %v", err)
 	}
 
 	updated := strings.Replace(string(initial), "# Issues", "# Issues (local old header)", 1)
 	updated += "\n- Issue 2099-01-01 localtest: keep me\n    Priority: Low. Area: tests.\n    Description: local entry.\n    Next step: keep.\n"
-	if err := os.WriteFile(issuesPath, []byte(updated), 0o644); err != nil {
+	if err := os.WriteFile(issuesPath, []byte(updated), 0o600); err != nil {
```

golangci-lint (2.12.2)

[high] G703: Path traversal via taint analysis

(gosec)